plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogConfig.java

```diff
@@ -67,4 +88,84 @@ public IcebergNessieCatalogConfig setDefaultWarehouseDir(String defaultWarehouse
         this.defaultWarehouseDir = defaultWarehouseDir;
         return this;
     }
+
+    @MinDuration("1ms")
+    public Duration getReadTimeout()
+    {
+        return readTimeout;
+    }
+
+    @Config("iceberg.nessie-catalog.transport.read-timeout")
```

Comment:
Just `iceberg.nessie-catalog.read-timeout` without "transport" is fine for me. Same for others.

Reply:
I think `transport` makes more sense here (same convention as Nessie catalog).  Else we don't know 

Also It was not there before. Added after a comment from @findinpath https://github.com/trinodb/trino/pull/17725#discussion_r1276291489

Reply:
Please take a look at the existing properties:
```sh
git grep @Config | grep transport
core/trino-main/src/main/java/io/trino/server/InternalCommunicationConfig.java:    @ConfigDescription("Enable the HTTP/2 transport")
```

Reply:
ok. Removed the transport keyword as suggested.